include/boost/gil/image_processing/harris.hpp

```diff
@@ -57,10 +57,10 @@ void compute_harris_responses(
             float dxdy = 0;
             float ddyy = 0;
             for (gil::gray32f_view_t::coord_t y_kernel = 0;
-                y_kernel < window_length;
+                y_kernel < static_cast<std::ptrdiff_t>(window_length);
```

Comment:
Again, `auto const window_length` could become `std::ptrdiff_t const window_length`

Reply:
Ok, did both changes. I rebased it again, but I realized it after pushing that I should have merged.

Reply:
@Scramjet911 No worries, rebase is fine too.